### PR TITLE
Command Line Arguments differences and alternatives between 2.5 and 2.8.

### PR DESCRIPTION
**Pull request by dacolgit**
added Command Line Arguments differences and alternatives between 2.5 and 2..8.2 in migration doc

### CODE REVIEW

File: docs/modules/publisher-migrationpath.md [hunk 1]
```diff
@@ -55,6 +55,78 @@ Please consider migrating your old published nodes JSON files that use `NodeId`
...
+| --ol, --opcmaxstringlen=VALUE           |  yes                     |                 |
+| --ot, --operationtimeout=VALUE          |  yes                     |                 |
+| --oi, --opcsamplinginterval=VALUE       |  yes                     |                 |
+| --op, --opcpublishinginterval=VALUE     |  yes                     |                 |
+| --ct, --createsessiontimeout=VALUE      |  yes                     |                 |
+| --ki, --keepaliveinterval=VALUE         |  yes                     |                 |
+| --kt, --keepalivethreshold=VALUE        |  yes                     |                 |
+| --aa, --autoaccept                      |  yes                     |                 |
+| --tm, --trustmyself=VALUE               |  yes                     |                 |
+| --to, --trustowncert                    |  no                      | same as trustmyself                |
```
Comment (karok2m) [inline docs/modules/publisher-migrationpath.md]: Can you please add `--` in front of command line options mentioned in `Alternative` column ?

Comment (dacolgit) [reply docs/modules/publisher-migrationpath.md]: added

File: docs/modules/publisher-migrationpath.md [hunk 2]
```diff
@@ -55,6 +55,78 @@ Please consider migrating your old published nodes JSON files that use `NodeId`
 
 To learn more about how to use comman-line arguments to configure OPC Publisher, please refer to [this](publisher-commandline.md) doc.
 
+## OPC Publisher 2.5.x Command Line Arguments supported in 2.8.2
+
+The following table describes the command line arguments, which were available in OPC Publisher 2.5.x and their compatibility in OPC Publisher 2.8.2.
```
Comment (karok2m) [inline docs/modules/publisher-migrationpath.md:60]: Can you please add a link to `publisher-commandline.md` saying that the full list of command line options for both versions is available there ?

Comment (dacolgit) [reply docs/modules/publisher-migrationpath.md:60]: The link is already present in the paragraph above "Command Line Arguments", It's just a couple of lines above

Comment (karok2m) [reply docs/modules/publisher-migrationpath.md:60]: Ah, great. Didn't see it. Thanks.

**Review by karok2m — APPROVED** [review]

**Pull request merged**